contrib/devtools/clog/main.go

```diff
@@ -141,36 +119,43 @@ func main() {
 		verboseLog.SetPrefix(logPrefix)
 	}
 
+	config = mustFindAndReadConfig(checkGetcwd())
 	if err := RootCmd.Execute(); err != nil {
 		log.Fatal(err)
 	}
 }
 
 func addEntryFileFromConsoleInput() error {
-
 	reader := bufio.NewReader(os.Stdin)
-	fmt.Println("Please enter the section (either: \"breaking\", \"features\", \"improvements\", \"bugfixes\")")
+	fmt.Fprintf(os.Stderr, "Please enter the section %v: ", config.Sections)
```

Comment:
This needs to be pretty-fied. Currently:

```
Please enter the section map[breaking:Breaking Changes bugfixes:Bugfixes features:New Features improvements:Improvements]:
```

Reply:
Yes, forgot to replace that with the list of map's keys